src/Rule/IsArray.php

```diff
@@ -10,6 +10,19 @@
  */
 class IsArray implements ValidationRuleInterface
 {
+    /**
+     * @var string | null
+     */
+    private $message;
+
+    /**
+     * IsArray constructor.
+     * @param string|null $message
+     */
+    public function __construct(string $message = null)
```

Comment:
Please add make the type nullable: 
```php
?string $message = null
```

Reply:
Fixed.
https://github.com/Morebec/Validator/pull/16/files/c32f74bf3b00242ac0c7a3f5b96800c1e49063e6..b4a69c683baec88f7afc0ba526d6f48ea801a5f0